src/evm/initialize-provider.test.ts

```diff
@@ -0,0 +1,123 @@
+import { ethers } from 'ethers';
+import { initializeProvider } from './initialize-provider';
+import { BASE_FEE_MULTIPLIER, PRIORITY_FEE_IN_WEI } from '../constants';
+import { ChainConfig } from '../types';
+
+describe('initializeProvider', () => {
+  beforeEach(() => jest.restoreAllMocks());
+
+  const chain: ChainConfig = {
+    maxConcurrency: 100,
+    authorizers: [],
+    contracts: {
+      AirnodeRrp: '0x5FbDB2315678afecb367f032d93F642f64180aa3',
+      RrpBeaconServer: '0xB7f8BC63BbcaD18155201308C8f3540b07f84F5e',
+      DapiServer: '0x9fE46736679d2D9a65F0992F2272dE9f3c7fa6e0',
+    },
+    id: '31337',
+    providers: { local: { url: 'http://127.0.0.1:8545' } },
+    type: 'evm',
+    options: {
+      txType: 'eip1559',
+      baseFeeMultiplier: '2',
```

Comment:
The gas price types will change to numbers in airnode v0.5, but these will be updated in the refactoring PRs.